xbmc/cores/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -328,12 +328,27 @@ COverlay* CRenderer::Convert(CDVDOverlaySSA* o, double pts)
 {
   CRect src, dst;
   g_renderManager.GetVideoRect(src, dst);
-
-  int width  = MathUtils::round_int(dst.Width());
-  int height = MathUtils::round_int(dst.Height());
-
+  RESOLUTION_INFO res = g_graphicsContext.GetResInfo();
+
+  int frameWidth = res.iWidth;
```

Comment:
@FernetMenta use is because libass name it as frame. so change the "frame" to "screen"? or just add comment lik this:
```
// libass render by frame, the frame size including margins between video to screen edge.
// so libass allow to render subtitles into the margins. this has been used to show subtitles in
// the top or bottom "black bar" between video to screen border
```